web-marketplace/src/hooks/projects/useProjectsWithOrders.ts

```diff
@@ -241,6 +247,8 @@ export function useProjectsWithOrders({
   });
 
   return {
+    allProjects: projectsWithOrderData,
+    haveOffChainProjects: allOffChainProjects?.length > 0,
```

Comment:
we need this now in order to display in the projects filter only the credit classes that have projects and the "unregistered projects" filtered only if there are approved off chain projects